backend/server.js

```diff
@@ -44,12 +44,34 @@ app.use(userRoutes);
 
 //GETS
 app.get("/players", async (req, res) => {
-  const { team, orderParam, ascending } = req.query;
+  const { sortType } = req.query;
+  const [feature, ascending] = sortType.split(" ");
+
+  const players = await prisma.player.findMany({
+    orderBy: [
+      {
+        [feature]: ascending,
+      },
+    ],
+  });
 
```

Comment:
<img width="1099" alt="Screenshot 2024-07-07 at 11 56 06 PM" src="https://github.com/GloriaXM/numbers-game/assets/112780860/0fe7dfa3-f39c-4b4c-9fcb-ac6681c2d859">
sortType will be passed in through the id of the frontend sort button. We predefine possible sortType values (and thus sort button id values) to <feature_name> + "  " + <asc or desc> to pass directly into prisma